keel-sql/src/main/kotlin/com/netflix/spinnaker/keel/sql/SqlDeliveryConfigRepository.kt

```diff
@@ -983,6 +983,29 @@ class SqlDeliveryConfigRepository(
     }
   }
 
+  override fun triggerRecheck(application: String) {
```

Comment:
one small (maybe improvement) - you can pass the delivery config itself here from the caller functions (all of them already fetching it) instead of fetching it by app name.

Reply:
We need the UID of the delivery config because that's what the checked at table uses. I'm not loading the entire delivery config here. And we have to fetch that UID because it doesn't come with the delivery config object :(

Reply:
ohhh in that case, forget it 😅

Reply:
I think @gal-yardeni's suggestion is legit. In fact, I think we should standardize on passing the objects to internal APIs instead of keys, as this kind of sub-optimal, redundant look up is all over the place. We could just add `uid` with a `@JsonIgnore` to `DeliveryConfig`. We already do that for `ConstraintState`, as an example.

Reply:
@luispollo I think that would be better done as a separate change, but I agree with you and Gal. Seems like it would be an improvement.